pages/plugins.vue

```diff
@@ -1,20 +1,81 @@
 <template>
   <div class="container">
     <PluginsNav />
-    <div class="tutorial-markdown-window">
-      <Plugins />
+    <div class="family-grid-wrapper">
+      <h1 class="hapi-header">Plugins</h1>
+      <p>
+        There are dozens of plugins for hapi, ranging from documentation to authentication,
+        and much more. If you wrote or use a plugin that you don't see on this list, please
+        send us a <a href="https://github.com/hapijs/hapi.dev/blob/master/static/lib/plugins.md">pull request</a>.
```

Comment:
I think you'll want to change this link to the new json file right?

Reply:
fixed